### PR TITLE
Implement test fixes for IMA policy overhaul

### PR DESCRIPTION
**Pull request by mbestavros**
For Keylime PR https://github.com/keylime/keylime/pull/1147.

### CODE REVIEW

**Comment by kkaarreell** [comment]
Hi @mbestavros , I believe that failures we do see here are related to the recently introduced change - tmt dynamic ref evaluation. We have it in place so we can better maintain and run tests for various distros and releases from respective branches, however it seems to conflict with Packit CI. I am currently trying to find out how to fix it.

**Comment by kkaarreell** [comment]
Mark, I have disabled dynamic ref evaluation by default in keylime-tests repository, please rebase once more. However, tests are now failing anyway due to some change in your keylime PR.

**Review by kkaarreell — APPROVED** [review]
Thank you very much for these test updates!
Please, once your keylime PR is ready, undo changes in main.fmf and squash all commits into one so we can merge it.

**Review by Koncpa — APPROVED** [review]

File: plans/main.fmf
Comment (kkaarreell) [inline plans/main.fmf:6]: Just so we don't forget, this file should be restored to its current version before the final merge.

**Comment by mbestavros** [comment]
@kkaarreell Mirroring what I've done on https://github.com/keylime/keylime/pull/1147, I've added a separate final commit that contains the updated branch info. We can drop this commit before merge as needed.

**Comment by kkaarreell** [comment]
Hi @mbestavros , could you please rebase? There were no major changes in tests but I want to be sure we are OK to merge it so we won't be blocking merge in upstream eventually (or further PR testing).

**Comment by mbestavros** [comment]
@kkaarreell rebased!

**Comment by kkaarreell** [comment]
Thank you. Btw, if I understand this change properly the old way was deprecated but should be usable for some time. I am planning to create a test test will be using the old approach so we can see it was not broken.

**Pull request merged**
